manifests/state-spectrum-x-operator/0010-spectrumx-operator.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: spectrum-x-operator
+  namespace: {{ .RuntimeSpec.Namespace }}
+  labels:
+    control-plane: spectrum-x-operator
+    app.kubernetes.io/name: spectrum-x-operator
+spec:
+  selector:
+    matchLabels:
+      control-plane: spectrum-x-operator
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: spectrum-x-operator
+      labels:
+        control-plane: spectrum-x-operator
+    spec:
+      tolerations:
+        {{- if .Tolerations }}
+        {{- .Tolerations | yaml | nindent 8 }}
+        {{- end }}
+        - key: nvidia.com/gpu
+          operator: Exists
+          effect: NoSchedule
+      securityContext:
+        runAsNonRoot: true
+        # TODO(user): For common cases that do not require escalating privileges
+        # it is recommended to ensure that all your Pods/Containers are restrictive.
+        # More info: https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted
+        # Please uncomment the following code if your project does NOT have to work on old Kubernetes
+        # versions < 1.19 or on vendors versions which do NOT support this field by default (i.e. Openshift < 4.11 ).
+        # seccompProfile:
+        #   type: RuntimeDefault
+      {{- if .CrSpec.ImagePullSecrets }}
+      imagePullSecrets:
+      {{- range .CrSpec.ImagePullSecrets }}
+      - name: {{ . }}
+      {{- end }}
+      {{- end }}
+      containers:
+        - command:
+            - /manager
+          args:
+            - --leader-elect
+            - --health-probe-bind-address=:8081
+            - --cm-namespace={{ .RuntimeSpec.Namespace }}
+            - --cidrpools-namespace={{ .RuntimeSpec.Namespace }}
```

Comment:
Add config map name, if added it to the API `--cm-name`

Reply:
done